### ISSUE TITLE
Monthly lookup quota is incorrect

### ISSUE THREAD

**Issue by Kreeblah** [Jerry23011/VirusTotal-macOS#3]
I have a free account with VirusTotal, and they show the following when I log in:

<img width="617" alt="Screenshot 2024-06-11 at 21 32 09" src="https://github.com/Jerry23011/VirusTotal-macOS/assets/1812790/d09c3967-4833-4ad1-899e-5bed43d16000">

For monthly API calls, I have a limit of 15.5K.  However, when I look in the application here, I see this:

<img width="584" alt="Screenshot 2024-06-11 at 21 32 16" src="https://github.com/Jerry23011/VirusTotal-macOS/assets/1812790/9419cce9-757a-45bb-96b3-18eb97ea4a1f">

For some reason, it's detecting a monthly quota of 1,000,000,000.

**Comment by Jerry23011** [comment]
Yes, it's showing 1,000,000,000 because that's the returned value from the API.

Since all six values are handled together, it would make the code more complex by setting the monthly maximum quota by itself since the monthly quota is 15.5K, which is 500 daily quota * 31 days. It's basically "unlimited" as long as you don't exceed the daily quota.

Another consideration of using the API returned value directly is that VT sometimes changes its quota. The one on their website is outdated, as there is no longer a minute quota but instead a new hourly quota. You can try it yourself by sending more than 4 requests/min.

I do think it may cause confusion, and I will try to handle that in the future.

**Comment by Kreeblah** [comment]
Oh, I didn't realize their actual limits were different than what their site says.  I did notice the hourly quota in the application there, but since that came out to the same as 4 requests/minute but spread over an hour, I didn't give it much thought.

But, yeah, with their monthly limit being that high, there's really no way to hit that on the free plan.

Thank you for creating this application, by the way, and for making it open source.  It's really helpful.

**Comment by Jerry23011** [comment]
Great to hear that you are enjoying it :)

I’ll leave this open and see what can be done
